docs/platform/howto/create-custom-cloud.rst

```diff
@@ -0,0 +1,47 @@
+Create a custom cloud in Aiven |beta|
+=====================================
+
+Learn how to add a custom cloud to your Aiven administrator's account and check out how it works.
```

Comment:
We're creating the custom cloud at the Organization level and de-emphasizing the notion of the account.  Admin is correct though.